MaterialLib/SolidModels/Lubby2.cpp

```diff
@@ -94,13 +94,15 @@ Lubby2<DisplacementDim>::integrateStress(
     // calculation of deviatoric parts
     auto const& P_dev = Invariants::deviatoric_projection;
     KelvinVector const epsd_i = P_dev * eps;
+    KelvinVector const epsd_t = P_dev * eps_prev;
 
-    // initial guess as elastic predictor
+    // initial guess as elastic predictor. Note: dimensionless stresses!
```

Comment:
`sigma_prev` two lines below is not dimensionless.

Reply:
mis description, sigd_t is the dimensionless stress...